lib/elements/confirm.js

```diff
@@ -33,6 +33,7 @@ class ConfirmPrompt extends Prompt {
   }
 
   submit() {
+    this.value = this.value !== void 0 ? this.value : false;
```

Comment:
Value will also always be a boolean, so this is always submitting false.

Instead you might want

```js
this.value = this.value || false;
```